index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+      <header id="header">
+        <div class="logo">
+          <img
+            src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png"
+            alt="logo"
+            id="header-img"
+          />
+        </div>
+        <nav id="nav-bar" content="width=device-width, initial-scale=1.0">
+          <ul>
+            <li>
+              <a href="#features" class="nav-link"></a>
+            </li>
+            <li>
+              <a href="#how-it-works" class="nav-link"></a>
+            </li>
+            <li>
+              <a href="#pricing" class="nav-link"></a>
+            </li>
+          </ul>
+        </nav>
+      </header>
+      <div class="container"></div>
+      <section class="products">
+        <h2>Hand Crafted, home made masterpices</h2>
+        <form id="form" action="https://www.freecodecamp.com/email-submit">
+          <input
+            name="email"
+            id="email"
+            type="email"
+            placeholder="Enter your email address"
+            required=""
+          />
+          <input id="submit" type="submit" value="Get Started" class="button" />
+        </form>
+      </section>
+      <article class="p-details">
+        <section id="features">
+          <div class="grid">
+            <div class="details">
+              <h2>Premium Materials</h2>
+              <p>
+                Our trombones use the shiniest brass which is sourced locally.
+                This will increase the longevity of your purchase.
+              </p>
+            </div>
+          </div>
+          <div class="grid">
+            <div class="details">
+              <h2>Fast Shipping</h2>
+              <p>
+                We make sure you recieve your trombone as soon as we have
+                finished making it. We also provide free returns if you are not
+                satisfied.
+              </p>
+            </div>
+          </div>
+          <div class="grid">
+            <div class="details">
+              <h2>Quality Assurance</h2>
+              <p>
+                For every purchase you make, we will ensure there are no damages
+                or faults and we will check and test the pitch of your
+                instrument.
+              </p>
+            </div>
+          </div>
+        </section>

```

Comment:
# Required Changes ♻️

Kindly, I suggest you implement FLEXBOX and other styles in ID "features", this detail so that your site looks more like accurate alignment